sleap/config/labeled_clip_form.yaml

```diff
@@ -18,6 +18,10 @@ main:
     label: Use GUI Visual Settings (colors, line widths)
     type: bool
     default: true
+  - name: background
+    label: Video Background
+    type: list
+    options: Original,Black,White,Grey
```

Comment:
These options should match case sensitivity, unless we handle this in visuals.py.

Reply:
Sure I'll make it lowercase in the docstrings and the option for GUI. 
We could keep it in lowercase to be consistent.